en_US/getting-started/getting-started.md

```diff
@@ -0,0 +1,250 @@
+- Data persistence and bridging
+- Management & monitoring center
+- Global technical support team
+
+[Try Free](https://www.emqx.com/en/try?product=enterprise)
+:::
+
+::::
+
+## Install EMQ X
+
+### Deploy in EMQ X Cloud
+
+EMQ X Cloud is the first fully managed MQTT 5.0 public cloud service in the world. With the support of EMQ X Cloud, you can create an EMQ X cluster on the cloud and use all the features of EMQ X Enterprise. This allows you to spend more time on business connections and less time for EMQ X operation, maintenance, and management.
+
+- [Create and login EMQ X cloud account](https://docs.emqx.com/en/cloud/latest/quick_start/introduction.html)
+- [Create a free trial deployment](https://docs.emqx.com/en/cloud/latest/quick_start/create_free_trial.html)
+
+### Running EMQ X in containers
+
+Learn more about the docker image on [Docker Hub](https://hub.docker.com/r/emqx/emqx). Container deployment is one of the ways you can quickly start to experiment with EMQ X
+
+1. Get the Docker image
+
+```bash
+docker pull emqx/emqx:latest
+```
+
+1. Start Docker container
+
+```bash
+docker run -d --name emqx -p 1883:1883 -p 8081:8081 -p 8083:8083 -p 8084:8084 -p 8883:8883 -p 18083:18083 emqx/emqx:latest
+```
+
+For more information about Docker installation and building a cluster quickly with Docker Compose, please refer to [Running with Docker (including a simple docker-compose cluster)](./install.md#install-emq-x-in-docker-contain-a-simple-docker-compose-cluster).
+
+### Install EMQ X on Kubernetes
+
+For users using Kubernetes, EMQ X provides [EMQ X Kubernetes Operator](https://www.emqx.com/en/emqx-kubernetes-operator). This Operator can help you quickly deploy an EMQ X cluster with complete functions that can be used in a production environment in Kubernetes environment.
+
+EMQ X Kubernetes Operator is an application orchestration tool developed based on Kubernetes native API, which is used for automatic deployment and lifecycle management of EMQ X clusters. You can refer to the [documentation](https://github.com/emqx/emqx-operator/blob/main/docs/user-guides/get-started.md) of the EMQ X Kubernetes Operator to learn how to deploy EMQ X using the Operator.
+
+### Install EMQ X on Terraform
+
+Deploy all infrastructure including EMQ X Enterprise clusters on the mainstream public cloud with one click through Terraform. The fastest way to deploy EMQ X Enterprise clusters on the public cloud：
+
+- [Deploy on AWS](https://github.com/emqx/tf-aws)
+- [Deploy on Alibaba Cloud](https://github.com/emqx/tf-alicloud)
+
+For more information about Terraform installation and deployment, please refer to [EMQ X Terraform](https://www.emqx.com/en/emqx-terraform) .
+
+### Run in a virtual machine or physical machine
+
+EMQ X Enterprise can be directly deployed on bare metal servers or virtual machines. A minimum of 2 core and 4GB machine can run the EMQ X Enterprise. It can support CentOS/Debian/Ubuntu/MacOS operating systems.
```

Comment:
```suggestion
EMQ X Enterprise can be deployed directly to bare metal servers or virtual machines. A minimum of 2 core and 4GB machine can run the EMQ X Enterprise. It can support CentOS/Debian/Ubuntu/MacOS operating systems.
```